### PR TITLE
Feature/vps backup storage order

### PR DESCRIPTION
**Pull request by JDownloader**

### CODE REVIEW

File: client/app/vps/backup-storage/order/vps-order-backup-storage.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<section class="container-fluid cui-page__content cui-page__content-flex">
+
+    <div class="row d-lg-flex">
+        <div class="col-xm-12">
+            <p>OPT:{{$ctrl.model.optionDetails}}</p>
```
Comment (Mboulianne) [inline client/app/vps/backup-storage/order/vps-order-backup-storage.html]: hardcoded OPT?  Will this be meaningful for every language?

File: client/app/vps/dashboard/vps-dashboard.controller.js [hunk 2]
```diff
@@ -66,6 +66,7 @@ class VpsDashboardCtrl {
         this.VpsService.getTabSummary(this.serviceName, true)
             .then(summary => {
                 this.summary = summary;
+                this.initBackupStorageActions();
```
Comment (Mboulianne) [inline client/app/vps/dashboard/vps-dashboard.controller.js]: I don't think this needs to be done in .then ;).

It's a detail tho.

Comment (Mboulianne) [reply client/app/vps/dashboard/vps-dashboard.controller.js]: also, if action availability is dependant of this.loaders.summary = false I'd expect to see it in action.isAvailable

Comment (JDownloader) [reply client/app/vps/dashboard/vps-dashboard.controller.js]: I wanted to load all actions at the same place in the controller. BackupStorageAction is not using the summary but SnapshotActions is. It's a case of a controller who do more than one thing, we should check to separate functionalities...

File: client/app/vps/backup-storage/order/vps-order-backup-storage.html (hunk 1 above)
Comment (JDownloader) [reply client/app/vps/backup-storage/order/vps-order-backup-storage.html]: I cleary forget about this, I will remove it

File: client/app/vps/veeam/order/vps-order-veeam.controller.js [hunk 3]
```diff
@@ -30,7 +30,6 @@ class VpsOrderVeeamCtrl {
                     });
             })
             .catch(data => {
-                $scope.resetAction();
                 $scope.setMessage($translate.instant("vps_configuration_veeam_order_step1_loading_error"), data);
```
Comment (whispyy) [inline client/app/vps/veeam/order/vps-order-veeam.controller.js]: $scope.message remplacer par this.cloudmessage.error

File: client/app/vps/dashboard/vps-dashboard.controller.js (hunk 2 above)
Comment (JDownloader) [reply client/app/vps/dashboard/vps-dashboard.controller.js]: I working on a sepatate branch on this issue, I will create a new service to provide all actions for the dashboard.

**Pull request merged**